### PR TITLE
fix(#1285): prevent html table events from being lost when using goa-…

### PR DESCRIPTION
**Pull request by chrisolsen**
…table

### CODE REVIEW

**Review by ArakTaiRoth — DISMISSED** [review]

File: libs/web-components/src/components/table/Table.svelte [hunk 1]
```diff
@@ -90,80 +91,31 @@
...
-    <tbody><tr><td /></tr></tbody>
-    <tfoot><tr><td /></tr></tfoot>
-  </template>
-</table>
+  `}
+>
+  {#if _isTableRoot}
+    <slot />  
+  {:else}
+    <table style="width: 100%;">
```
Comment (vanessatran-ddi) [inline libs/web-components/src/components/table/Table.svelte:106]: It seems that the style here override the "variant" attribute of a table.

Comment (chrisolsen) [reply libs/web-components/src/components/table/Table.svelte:106]: Can you better explain what is being overridden? The `variant` is `relaxed` and `sticky` and neither of those classes have a `width` style attribute.

Comment (vanessatran-ddi) [reply libs/web-components/src/components/table/Table.svelte:106]: Yes I tried on the storybook, looks like the table width is bigger. 
image

I see the current storybook, the table seems to have class name "normal"

File: libs/web-components/src/components/table/Table.svelte [hunk 2]
```diff
@@ -90,80 +91,35 @@
...
-  </template>
-</table>
+  `}
+>
+  {#if _isTableRoot}
+    <slot />
+  {:else}
+    <table
+      style={
+       width && "width: 100%;"
```
Comment (vanessatran-ddi) [inline libs/web-components/src/components/table/Table.svelte:108]: 👍

**Review by vanessatran-ddi — APPROVED** [review]

**Review by ArakTaiRoth — APPROVED** [review]

**Pull request merged**